modules/installation/pages/install-server-unified.adoc

```diff
@@ -4,20 +4,18 @@
 
 {susemgr} is a {suse} product within the {sle} product family.
 This section describes how to install {susemgr} Server from the {sle} installation media.
-For this topic we assume that you have already registered your {susemgr} product with {scc} and have obtained a registration code.
+For this topic we assume that you already have valid organization credentials with {scc} and have obtained a registration code for your {susemgr}.
 
 For information on registering with {scc}, retrieving your organization credentials from {scc}, or obtaining installation media, see xref:general-requirements.adoc[].
 
-[NOTE]
-====
-To change a {sles}{nbsp}{sles-version}{nbsp}{sp-version} into a {susemgr}{nbsp}{productnumber}, follow the instructions at xref:installation:install-vm.adoc[].
-====
 
 [NOTE]
 ====
-Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:hardware-requirements.adoc[Hardware].
+Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:hardware-requirements.adoc.
```

Comment:
```suggestion
Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:hardware-requirements.adoc[].
```
We usually put empty bracket at the end of an xref.  I do not know whether the output wiil be the same without brackets.

Reply:
Oh, it should have the book name, too.

```suggestion
Before installing {susemgr}, ensure your physical or virtual machine has enough disk space and RAM by checking the requirements at xref:installation:hardware-requirements.adoc[].
```

Reply:
I think the book name is optional for references within a module (or pages directory?)  Of course, for consistency an absolute xref is better.